fs/manifest.go

```diff
@@ -369,8 +369,8 @@ func (m *Manifest) AddFileTo(dir map[string]interface{}, filepath string, hostpa
 }
 
 // AddLibrary to add a dependent library
-func (m *Manifest) AddLibrary(path string) {
-	parts := strings.FieldsFunc(path, func(c rune) bool { return c == '/' })
+func (m *Manifest) AddLibrary(targetroot string, path string) {
+	parts := strings.FieldsFunc(strings.TrimPrefix(path, targetroot), func(c rune) bool { return c == '/' })
```

Comment:
The Manifest type has a targetRoot field, so I would use that (as in `strings.TrimPrefix(path, m.targetRoot)`) instead of adding a parameter to AddLibrary().

Reply:
Thanks for pointing that out!